src/pages/dashboard/components/account-info/account-info.component.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
+import {
+  Divider,
+  Grid,
+  CircularProgress,
+  Paper,
+  Box,
+  Typography,
+} from "@mui/material";
+import { users } from "../../../../data/user";
```

Comment:
It's sometimes preferable to use the complete path to the file. Like in this case

Reply:
DOne